CMakeLists.txt

```diff
@@ -35,10 +35,10 @@ endif()
 # Enable C++11 support in all compilers. SymEngine will not compile unless
 # the C++11 support is enabled.
 if (CMAKE_CXX_COMPILER_ID MATCHES "GNU|Clang|Intel")
-    set(CXX11_OPTIONS "-std=c++11")
+    set(CXX11_OPTIONS "-std=c++14")
```

Comment:
Is this necessary?

Reply:
No. I only needed it for `make_unique`, which I will find some workaround for.

Reply:
Fixed, I switched back to C++11.